vm_supervisor/metrics.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import os
+from os.path import join
+from typing import Optional, Iterable
+from uuid import UUID
+
+from sqlalchemy import Column, Integer, String, Float, DateTime
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from .conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+Base = declarative_base()
+session_maker: Optional[sessionmaker] = None
```

Comment:
The idiomatic name for this is `Session` in the docs. You can also create it directly from this level, I don't see the benefit of `get_database_sessionmaker`.

Reply:
More info: https://docs.sqlalchemy.org/en/13/orm/session_basics.html#what-does-the-session-do